src/penn_chime/models.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, p: Parameters):
         }
 
         # Note: this should not be an integer.
-        # We're appoximating infected from what we do know.
+        # We're approximating infected from what we do know.
```

Comment:
can you un-piggyback this please? i don't want anything on the dash branch to touch the `penn_chime` module, to reduce MC